src/main/java/org/quiltmc/loader/impl/discovery/ClasspathModCandidateFinder.java

```diff
@@ -104,6 +70,37 @@ public static void findCandidatesStatic(ModAdder out) {
 		}
 	}
 
+	private static void findCandidates(String metadataName, Map<Path, List<Path>> pathGroups, ModAdder out) throws IOException {
+		Enumeration<URL> quiltMods = QuiltLauncherBase.getLauncher().getTargetClassLoader().getResources(metadataName);
+		while (quiltMods.hasMoreElements()) {
+			URL url = quiltMods.nextElement();
+
+			try {
+				Path path = LoaderUtil.normalizeExistingPath(UrlUtil.getCodeSource(url, metadataName));
+				List<Path> paths = pathGroups.get(path);
+
+				if (paths == null) {
+                    if (!url.getProtocol().equals("jar")) {
```

Comment:
There's a few places like this were you've used spaces instead of tabs - did you setup your environment again? (THis isn't actually a problem though, just something we'll replace at some point I guess?)